packages/showcases/css/.storybook/main.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  stories: [
+    '../stories/*.stories.(mdx|js)',
+    '../stories/guidelines/**/*.stories.(mdx|js)',
+    '../stories/components/**/*.stories.(mdx|js)',
```

Comment:
I imagine that's not a good practice to put that ? 
```suggestion
    '../stories/**/*.stories.(mdx|js)',
```

Reply:
I've separated because we need to see introduction, then guidelines, then components in this order :)

Reply:
Great thanks a lot 🚀